index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="EN">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Abhaya+Libre:wght@700&family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Inter&family=Poppins:wght@100&family=Roboto:wght@100;700&display=swap"
+     rel="stylesheet">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Poppins:wght@100;200;400&family=Roboto:wght@100;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Inter&family=Poppins:wght@100&family=Roboto:wght@100;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Inter&family=Poppins:wght@100&family=Roboto:wght@100;700&display=swap" 
+    rel="stylesheet">
+    <title>Riyaz's Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="site-header">
+        <div id="title-text"><h3>
+          <a class="logo-text" href="#">Riyaz Ahmed</a>
```

Comment:
The requirement for this part is to use the provided font which is `Roboto`. Kindly use it.

Reply:
Done as advised.